bin/src/main.rs

```diff
@@ -296,19 +320,14 @@ async fn _main() {
 
     let fs_source = tail::RestartingTailer::new(
         ds_source_params,
-        |item| match item {
-            Err(fs::cache::Error::WatchOverflow) => {
-                warn!("overflowed kernel queue, restarting stream");
-                true
-            }
-            _ => false,
-        },
+        // TODO check for any conditions that require the tailer to restart
```

Comment:
does not look good to drop error processing.

Reply:
Yup, that's one of the parts that's still WIP, need to propagate the rescan event from the notify stream.